### PR TITLE
Add SENet to the Fabrik model zoo

### PR DESCRIPTION
**Pull request by idealadarsh**
Add ```SENet (Squeeze-and-Excitation Networks)``` an image classification model to the Fabrik model zoo.

Viualise: http://fabrik.cloudcv.org/caffe/load?id=20180106091323ectck

Source: https://github.com/hujie-frank/SENet

Related GCI Task: https://codein.withgoogle.com/dashboard/task-instances/6074491597750272/

### CODE REVIEW

File: docs/source/tested_models.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ List of all models for which import/export has been tested with Fabrik.
 * VGG-16 [\[Source\]](https://gist.github.com/ksimonyan/211839e770f7b538e2d8#file-readme-md)[\[Visualise\]](http://fabrik.cloudcv.org/caffe/load?id=20171208113208hjcvb)
 * DeepYeast [\[Source\]](http://kodu.ut.ee/~leopoldp/2016_DeepYeast/code/caffe_model/)[\[Visualise\]](http://fabrik.cloudcv.org/caffe/load?id=20180102135425bzkzy)
 * SpeechNet [\[Source\]](https://github.com/pannous/caffe-speech-recognition)[\[Visualise\]](http://fabrik.cloudcv.org/caffe/load?id=20180102135032ctsho)
+* SENet [\[Source\]](https://github.com/hujie-frank/SENet) [\[Visualise\]](http://fabrik.cloudcv.org/caffe/load?id=20180106091323ectck)
```
Comment (virajprabhu) [inline docs/source/tested_models.md]: Can you add a visualization link like it has been done for the others? You will need to load the model in Fabrik, and click "Share" to generate a URL.

Comment (virajprabhu) [reply docs/source/tested_models.md]: I see it's been done below – can you copy that over here to keep things consistent?

**Comment by virajprabhu** [comment]
Thanks @idealadarsh. Please look at comments.

**Comment by virajprabhu** [comment]
Also, this is a classification model right? Not captioning?

File: docs/source/tested_models.md (hunk 1 above)
Comment (idealadarsh) [reply docs/source/tested_models.md]: Here is the visualisation link: http://fabrik.cloudcv.org/caffe/load?id=20180106091323ectck

**Comment by idealadarsh** [comment]
Yes it is a classification model.

**Comment by utsavgarg** [comment]
Hi, please resolve merge conflicts and change the description to say it's a classification model, not a captioning one.

**Comment by idealadarsh** [comment]
@utsavgarg Resolved merge conflicts and changed the description.

**Comment by utsavgarg** [comment]
LGTM :)

**Pull request merged**